### ISSUE TITLE
Prettier is required peerDependency

### ISSUE THREAD

**Issue by gynekolog** [lukasoppermann/style-dictionary-utils#97]
I tried style-dictionary utils and got error `Cannot find package 'prettier' imported from pathtomyrepo/node_modules/.pnpm/style-dictionary-utils@4.0.0_style-dictionary@4.1.4/node_modules/style-dictionary-utils/dist/format/css-advanced.js`

So the prettier is required? Should I install prettier just for style-dictionary-utils?

**Comment by lukasoppermann** [comment]
> So the prettier is required? Should I install prettier just for style-dictionary-utils?

Yes, as this is used internally. It is just a peerDep so that you have more flexibility with the version

**Comment by gynekolog** [comment]
Should the installation instructions specify that prettier is required, right?:
`npm i -D style-dictionary-utils style-dictionary prettier`

Or would it be possible to remove the prettier dependency from the library? Since file formatting isn’t the primary purpose of style-dictionary-utils, it might be better to let users handle file formatting according to their preferences.

**Comment by lukasoppermann** [comment]
Prettier is used for output formatting. I think we do want to export formatted files from style dictionary.

Adding it to the install instructions is one option.

I guess we could also consider just adding it as a dependency until we run into issues.
